Firestore/Source/API/FSTUserDataConverter.mm

```diff
@@ -165,7 +166,7 @@ typedef NS_ENUM(NSInteger, FSTUserDataSource) {
   FSTUserDataSourceSet,
   FSTUserDataSourceMergeSet,
   FSTUserDataSourceUpdate,
-  FSTUserDataSourceQueryValue,  // from a where clause or cursor bound.
+  FSTUserDataSourceNonWrite,  // from a where clause, cursor bound, arrayUnion() element, etc.
```

Comment:
I'm not super-happy with "non-write" as a name for this concept.

How about "argument", "secondary", or something like that?

Reply:
I like FSTUserDataSourceArgument. Thanks.